platform/engineering/frontend/dependabot

```diff
@@ -0,0 +1,87 @@
+# Dependabot in `vets-website` repo
+
+**Author:** Jhonny Gonzalez
+
+**Last Updated:** June 25, 2020
+
+## Overview
+
+This is a decision making process to choose the most efficient automated dependency upgrade product.
+
+Based on this [evaluation of automated dependency upgrade products](https://github.com/department-of-veterans-affairs/va.gov-team/issues/9537). The final decision is to go with `Dependabot`
+
+## Objective
+
+To be able to rely on a automated dependency upgrade manager
+
+## Background
+
+`Dependabot` was initially installed in `vets-website` but it wasn't doing anything. There was no action by `Dependabot`, therefore, we started looking at alternatives.
+
+[This ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/9537) has an evaluation of Depfu, Renovate and Dependabot.
+
+`dependabot` has good settings and also [it's been integrated into GitHub](https://dependabot.com/blog/hello-github/).
+
+## Specifics
+
+`dependabot` uses a configuration file, `dependabot.yml`, that lives at the root of each repository to handle the dependencies for that repo.
+
+### `dependabot.yml` file located inside the `.github` folder
+
+```
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "daily"
+      time: "09:00"
+      timezone: "America/New_York"
+      # Allow up to 10 open pull requests for npm dependencies
+      open-pull-requests-limit: 10
+```
+
+## Resources
+
+- [Documentation](https://help.github.com/en/github/administering-a-repository/keeping-your-dependencies-updated-automatically)
+
+## Settings
+
+- `dependabot` will run everyday at 9am EST. 
+- `dependabot` will only create up to 10 pull request at the time.
+- `dependabot` can be located inside GitHub under the [Insigths tab > Dependency graph](https://github.com/department-of-veterans-affairs/vets-website/network/updates)
+- `dependabot` can be run manually from its settings tab inside GitHub
+
+## Procedures
+
+# Minor and patch
+
+Any minor or patch upgrades should be prioritized since they won't have major impact. However, even minor library version upgrades can break or insert inconsistencies in apps.
+
+In Minor or patches PRs, you need to still verify that there aren't any breaking changes.
+
+# Major
+
+A major version upgrade is often a more conscious decision to upgrade with the understanding that such an upgrade will require some work. Therefore, a ticket should be opened and extended work will be performed.
+
+## Failure after a PR is merged
+
+Truth is that tests are not perfect either and chances are you probably didn’t read the release notes looking for breaking changes before merging a green-light PR.
+
+If a PR merged is breaking the code, the PR should be reverted. A new PR should be created manually and a fix will need to be implemented along the upgrade to prevent any further issues.
```

Comment:
plain language suggestion 

```suggestion
If a dependency upgrade causes an issue not caught by automated testing, the upgrade should be quickly reverted. A new PR should be created manually and a fix will need to be implemented along the upgrade to prevent any further issues.

_Since automated tests may not catch all issues be sure to read release notes related to any dependency upgrade._
```